buddyboss/buddyboss-platform
# Lost Password link not working when WooCommerce plugin active

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #778.

### How to test the changes in this Pull Request:

1. Active WC
2. Login with any subscriber user
3. Go to Account -> Login Information
4. Find our link "Lost password" and click on it
And also check the same things as a view as

### Proof Screenshots or Video

https://www.loom.com/share/b9b06a04fa404aa2ab49c51e521eb163

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
